app/services/validation_beta_service.rb

```diff
@@ -269,4 +254,40 @@ def participants_yet_to_validate
   def chaser_scheduled?(profile)
     Delayed::Job.where("handler ILIKE ?", "%ParticipantDetailsReminderJob%#{profile.id}%").exists?
   end
+
+  def sit_with_unvalidated_participants_reminders
+    InductionCoordinatorProfile
+      .joins(schools: :ecf_participant_profiles)
+      .includes(schools: { active_ecf_participant_profiles: %i[ecf_participant_eligibility ecf_participant_validation_data] })
+      .where(
+        school_cohorts: {
+          cohort_id: Cohort.current.id,
+          induction_programme_choice: %w[core_induction_programme full_induction_programme],
+        },
+        ecf_participant_eligibility: {
+          participant_profile_id: nil,
+        },
+        ecf_participant_validation_data: {
+          participant_profile_id: nil,
+        },
+      ).distinct.find_each do |sit|
+        campaign = :unvalidated_participants_reminder
+
+        sign_in_url = Rails.application.routes.url_helpers.new_user_session_url(
+          host: Rails.application.config.domain,
+          **UTMService.email(campaign, campaign),
+        )
+
+        participant_validation_start_url = Rails.application.routes.url_helpers.participants_start_registrations_url(
+          host: Rails.application.config.domain,
+          **UTMService.email(campaign, campaign),
+        )
+
+        ParticipantValidationMailer.induction_coordinators_we_asked_ects_and_mentors_for_information_email(
```

Comment:
The usual question: IC with multiple schools?

If this email only asks them to log in, then perhaps seeing 5 of them is not a good look :D

Reply:
Since we are using `InductionCoordinatorProfile` and using a `distinct` at the end, this will only run once for each SIT, even if they have multiple schools.

Reply:
Aaah, good point!